tools/github_stats.py

```diff
@@ -176,13 +176,13 @@ def report(issues, show_urls=False):
     # Print summary report we can directly include into release notes.
     print('.. _github-stats:')
     print()
-    print('GitHub Stats')
+    print('GitHub statistics')
     print('============')
```

Comment:
These lengths need to match?
```suggestion
    print('=================')
```